client/lxd_network_acls.go

```diff
@@ -6,7 +6,24 @@ import (
 	"github.com/lxc/lxd/shared/api"
 )
 
-// CreateNetworkACL defines a new network ACL using the provided struct.
+// GetNetworkACLs returns a list of Network ACL structs.
+func (r *ProtocolLXD) GetNetworkACLs() ([]api.NetworkACL, error) {
```

Comment:
We'll want a `GetNetworkACLsNames` too for consistency.

Reply:
will do.